config/cspo/clusterstack.yaml

```diff
@@ -1,3 +1,16 @@
+apiVersion: infrastructure.clusterstack.x-k8s.io/v1alpha1
+kind: OpenStackClusterStackReleaseTemplate
+metadata:
+  name: cspotemplate
+  namespace: cluster
+spec:
+  template:
+    spec:
+      cloudName: "${CLOUD_NAME}"
```

Comment:
`cloudName` field has been removed in https://github.com/SovereignCloudStack/cluster-stack-provider-openstack/pull/101, hence it should not be here